bouncer/shared/gaslimit_ccm.ts

```diff
@@ -207,8 +208,14 @@ async function testGasLimitSwap(
   // This is a very rough approximation for the gas limit required. A buffer is added to account for that.
   if (minGasLimitRequired + BASE_GAS_OVERHEAD_BUFFER[destChain] >= gasLimitBudget) {
     let stopObservingCcmReceived = false;
+    console.log(
+      `${tag} Gas budget of ${gasLimitBudget} is too low. Expecting BroadcastAborted event. Time to wait for CCM event`,
+    );
 
-    await observeCcmReceived(
```

Comment:
I introduced a bug yesterday here (41953910eb5aaa8ca4c0d5f8e66d1980b94993b4), this was not meant to be awaited - but at least it's explicit now